advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,590 @@
+---
+title: 'Multiple Repositories'
+description: "Deeper look at the multiple repositories support feature"
+---
+
+The multiple repositories support was added in the [v2.33](https://github.com/pgbackrest/pgbackrest/releases/tag/release%2F2.33) release. One example where multiple repositories could be used, would be for situations where a local repository would have minimal retention (for fast restores), and a remote repository would have a longer retention (for redundancy and access across the enterprise).
+
+A new [`--repo`](https://pgbackrest.org/configuration.html#section-repository/option-repo) option has been added to set the repository for a command to operate on. All `repo*-` options are indexed to allow for configuring multiple repositories.
+
+In this section, we'll setup a **demo** cluster (using **EDB Postgres Advanced Server** 13 on CentOS 7) to demonstrate how each pgBackRest command reacts to this new feature.
```

Comment:
In this section, we will set up a **demo** cluster (using **EDB Postgres Advanced Server** version 13 on CentOS 7) to demonstrate how this new feature impacts each pgBackRest command.

Reply:
fixed in c2942c3